chassis/core/src/main/java/com/griddynamics/jagger/storage/fs/logging/ChronologyLogAggregator.java

```diff
@@ -20,19 +20,14 @@
 
 package com.griddynamics.jagger.storage.fs.logging;
 
+import com.google.common.io.Closeables;
 import com.griddynamics.jagger.storage.FileStorage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Required;
 
-import com.google.common.io.Closeables;
-
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Iterator;
-import java.util.PriorityQueue;
-import java.util.Set;
+import java.util.*;
```

Comment:
please, add setting in Idea to import all classes by name. No * imports allowed

Reply:
fixed